README.md

```diff
@@ -38,9 +38,11 @@ For instructions on building & testing these as well as setting up a development
 We welcome your contributions to this open source project!
 TLA⁺ is used in safety-critical systems, so we have a contribution process in place to ensure quality is maintained; read [CONTRIBUTING.md](CONTRIBUTING.md) before beginning work.
 
-Copyright History
+License & Copyright
 -----------------
-Copyright (c) 199?-2003 HP Corporation  
-Copyright (c) 2003-2020 Microsoft Corporation
+Copyright © 199?-2003 HP Corporation  
+Copyright © 2003-2023 Microsoft Corporation  
+Copyright © 2023-current Linux Foundation
```

Comment:
IANAL, but as far as I know, the original copyright doesn't end. It remains with HP and Microsoft, while any new contributions are copyrighted by the LF.